courses/machine_learning/deepdive2/structured/solutions/3c_bqml_dnn_babyweight.ipynb

```diff
@@ -0,0 +1,1152 @@
+{
```

Comment:
<ul><li>"Verify that" -&gt; "Run the following cells to verify that"</li><li>Link &nbsp;<a href="https://github.com/GoogleCloudPlatform/training-data-analyst/raw/9d258e061481c537a601c44e6210d04af4f2f5ca/courses/machine_learning/deepdive2/structured/solutions/1b_prepare_data_babyweight.ipynb" target="_blank">1b_prepare_data_babyweight</a> not working</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/670/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Done</p>

Reply:
<p>Done</p>